rules/windows/image_load/image_load_side_load_dbgmodel.yml

```
@@ -4,9 +4,9 @@ status: experimental
 description: Detects potential DLL sideloading of "DbgModel.dll"
 references:
     - https://hijacklibs.net/entries/microsoft/built-in/dbgmodel.html
-author: Gary Lobermier
+author: Gary Lobermier, modified by Josh Nickels
```

Small fp fixes do not qualify authorship, but instead are shown in the release credit.
```suggestion
author: Gary Lobermier
```